### PR TITLE
Easy problem of shopping cart

### PR DESCRIPTION
**Pull request by Anmolreshi**
Input from user:
input
output to user:
output

**Language Used** : _C++_ 
BASIC APPROACH 🔢 to solve the program :

- Used class in program

- Created object in main function to access class members functions.

- Used Time library to get Real Date , Time , Year and day of user current usage during program.

- added little bit systematic styles to program  for better view to user and easy understandings of the program to user.

### CODE REVIEW

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]
Please send a PR to `girlscript-blr:solution` branch. Kindly refer to [CONTRIBUTING.md](https://github.com/girlscript-blr/code-with-girlscript-bangalore/blob/master/CONTRIBUTING.md) file's Create Pull Request point.

Now that you have created the PR, you can click on the `Edit` button beside the PR title and change the branch this PR is pointing to. Please follow the below steps.

1. Click on `Edit`.
image

2. Change the base branch from `master` to `solution`.
image

Also. a brief description(not more than 4-5 lines/100 words) needs to be included containing the approach used for solving the problem along with the screenshot in a pdf document format.

The codes/files should be inside a single folder following a specific naming convention. The solution folder name should be `<your GitHub Id>_<your name>`. This new folder should be inside the `solutions` folder.

**Comment by Anmolreshi** [comment]
Done changes ! kindly Review

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]
Please create a folder with the name `<Your Github Id>_<Your name>` and put all the solutions there. Check out how others have added their solution from [here](https://github.com/girlscript-blr/code-with-girlscript-bangalore/tree/solution/Easy/1.%20Shopping%20Cart%20Problem/solutions).

**Comment by irshadjsr21** [comment]
Also, please don't make changes to your `master` branch. Create a new branch as stated in [CONTRIBUTING.md](https://github.com/girlscript-blr/code-with-girlscript-bangalore/blob/master/CONTRIBUTING.md) and add your solution there.

**Comment by Anmolreshi** [comment]
I CREATED CHANGES IN MASTER BRANCH OF MY FORKED REPOSITARY AND MADE FOLDER

File: Easy/1. Shopping Cart Problem/solutions/README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+Language Used : C++
+BASIC APPROACH 🔢 to solve the program :
+
+-> Used class in program 
+
+-> Created object in main function to access class members functions.
+
+-> Used Time library (<ctime>) to get Real Date , Time , Year and day of user current usage during program.
+
+-> Added little bit systematic styles to program for better view to user and easy understandings of the program to user.
```
Comment (irshadjsr21) [inline Easy/1. Shopping Cart Problem/solutions/README.md]: Please remove this change and add the solution approach inside your own folder (i.e. `Anmolreshi_anmolreshi`).

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]
Please create a PR from a new branch as stated in `CONTRIBUTIONG.md` file.

File: Easy/1. Shopping Cart Problem/solutions/README.md (hunk 1 above)
Comment (Anmolreshi) [reply Easy/1. Shopping Cart Problem/solutions/README.md]: okay

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]
Don't delete the `Easy/1. Shopping Cart Problem/solutions/README.md` folder. Keep it same as it was in the main repository without any changes.

Also, please don't make changes to your master branch. Create a new branch as stated in CONTRIBUTING.md and add your solution there.

**Comment by Anmolreshi** [comment]
> Don't delete the `Easy/1. Shopping Cart Problem/solutions/README.md` folder. Keep it same as it was in the main repository without any changes.
> 
> Also, please don't make changes to your master branch. Create a new branch as stated in CONTRIBUTING.md and add your solution there.
I have not deleted README file and i have done changes in master branch

**Comment by Anmolreshi** [comment]
I added my other branch name Solution to solution of girlscript its issue #47 kindly review it

**Pull request closed without merging**